languages/python/oso/polar/query.py

```diff
@@ -80,6 +80,7 @@ def handle_make_external(self, data):
         self.host.make_instance(cls_name, args, kwargs, id)
 
     def handle_external_call(self, data):
+        self.host.contains_partial = data["query_contains_partial"]
```

Comment:
@dhatch / @samscott89 are we fine with this temporary kludge shipping?

Reply:
What does this do?

Reply:
Since we now create partials on the host side (when encountering a Django `Manager` object), we need to ensure that we only do that when the user has already opted in to partials by calling `authorize_model` with a partial. If we create a partial when the user has not passed one in, we might end up spitting out a partial when they expect a boolean answer.

Reply:
I'm only seeing it used here in the Python code:
```
languages/python/oso/polar/query.py
83:        self.host.contains_partial = data["query_contains_partial"]

```
so I'm a little confused how it is doing anything?

Reply:
https://github.com/osohq/oso/pull/580/files#diff-f373f8ebee08aac2d42390cfdc23da6040052b9f6a5de87f57cca9d3ff245cc4R131

Reply:
https://github.com/osohq/oso/blob/89ac64c0567fb880456b2e89446e60eaef0694ea/languages/python/oso/polar/host.py#L130-L135

Reply:
Ignore me, missed that it was renamed